libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -5,14 +5,15 @@ import { of } from 'rxjs';
 import { catchError, concatMap, exhaustMap, map } from 'rxjs/operators';
 import { ReadingListItem } from '@tmo/shared/models';
 import * as ReadingListActions from './reading-list.actions';
+import { API_ENDPOINT } from '@tmo/books/data-access';
 
 @Injectable()
 export class ReadingListEffects implements OnInitEffects {
   loadReadingList$ = createEffect(() =>
     this.actions$.pipe(
       ofType(ReadingListActions.init),
       exhaustMap(() =>
-        this.http.get<ReadingListItem[]>('/api/reading-list').pipe(
+        this.http.get<ReadingListItem[]>(API_ENDPOINT.READING_LIST).pipe(
```

Comment:
What might be better than putting the endpoint into a constant?

Could you put the entire call into another service, perhaps?